### DISCUSSION TITLE
Running peacock on docker installation

### DISCUSSION THREAD

**Discussion by Bawino** [idaholab/moose#23973] [Q&A Getting Started]
Hello everyone,

I am trying to run Peacock on a Docker container in my Windows 11 machine. However, I am encountering an error that says "Peacock command not found". After some investigation, it appears that the path to the Peacock executable is not included in my environment variables.

I have been trying to locate the file where I can edit the path to include the Peacock executable, but so far, I have not been successful. Could anyone provide guidance on how to locate the file or files where I can edit the path to include the correct one for Peacock?

Thank you in advance for your help.

**Comment by milljm** [comment]
The Peacock executable will be located here (if you followed the directions to clone moose to `~/projects/moose`:
```bash
$HOME/projects/moose/python/peacock/peacock
```
In order to make the above path included in your PATH environment variable permanently, add the following to your shell startup profile:

Bash users:
```bash
echo "export PATH=\$PATH:\$HOME/projects/moose/python/peacock/" >> ~/.bash_profile
``` 
Zsh users:
```bash
echo "export PATH=\$PATH:\$HOME/projects/moose/python/peacock/" >> ~/.zshrc
``` 
To figure out what shell you use (Bash, Zsh, perhaps something else):
```bash
echo $0
```
If this returns something other than the two common shells I posted, you'll need to do some searching on the internet on how to modify your specific shell's startup profile.
